internal/utils/common.go

```diff
@@ -216,6 +222,26 @@ func Slice2Map(s []string) map[string]bool {
 	return m
 }
 
+func GenericSlice2Map[T string | int | any](t []T) booleanMap {
```

Comment:
how about turning `Slice2Map` into

```go
func Slice2Map[T comparable](t []T) map[T]bool {
	m := map[T]bool{}
	for _, item := range t {
		m[item] = true
	}
	return m
}
```

and calling it from `internal/configure/disks/disks.go` in this way ?

```go
hostExcludeMap := utils.Slice2Map(hostExclude)
// ...
if _, ok := hostExcludeMap[h]; !ok {
    // ...
}
```

Reply:
but interface is not `comparable`

Reply:
My bad, Go1.20 is required to do so.

Reply:
Done.

Reply:
assertion interface slice and then convert into string.
![image](https://user-images.githubusercontent.com/19409549/222341099-278fc04e-d387-4097-9b45-a53366e241c6.png)